indra/newview/llpuppetevent.h

```diff
@@ -40,6 +40,11 @@
 #include "v3math.h"
 #include "llframetimer.h"
 
+namespace LLIK {
+    size_t pack_vec3(U8* wptr, LLVector3 vec);
+    size_t pack_quat(U8* wptr, LLQuaternion quat);
+};
```

Comment:
The `pack_*()` methods were originally helper methods in **llpuppetevent.cpp**, but they are being used in **llvoavatarself.cpp** so I moved them to the `LLIK` namespace to make them available, but now I think they need a better home than `LLIK`.